sphinx_markdown_builder/contexts.py

```diff
@@ -288,7 +288,7 @@ def make(self):
 
 
 class IndentContext(SubContext):
-    def __init__(
+    def __init__(  # pylint: disable=too-many-positional-arguments
```

Comment:
This error is due to updates in pylint.
The correct fix is adding the following line to `pyproject.toml`.
```
max-positional-arguments = 6
```